### ISSUE TITLE
`string` is not assignable to type `Any`

### ISSUE THREAD

**Issue by NoahSaso** [cosmology-tech/telescope#438]
When generating `/cosmos.staking.v1beta1.staking.Validator`, the `consensusPubkey` type seems to get confused in the Amino encoders.

```ts
fromAmino(object: ValidatorAmino): Validator {
    return {
      operatorAddress: object.operator_address,
      consensusPubkey: encodeBech32Pubkey({
        type: "tendermint/PubKeySecp256k1",
        value: toBase64(object.consensus_pubkey.value)
      }, "cosmos"),
...
```

This causes the following error:
```
Type 'string' is not assignable to type 'Any'.ts(2322)
staking.ts(264, 3): The expected type comes from property 'consensusPubkey' which is declared here on type 'Validator'
(property) Validator.consensusPubkey: Any
```

`encodeBech32Pubkey` returns a string, but `Validator.consensusPubkey` is an `Any` protobuf. `object.consensus_pubkey` is `AnyAmino` which has a `value` of `any`.

The same thing happens in the `toAmino` function:
```ts
toAmino(message: MsgCreateValidator): MsgCreateValidatorAmino {
    const obj: any = {};
    obj.description = message.description ? Description.toAmino(message.description) : undefined;
    obj.commission = message.commission ? CommissionRates.toAmino(message.commission) : undefined;
    obj.min_self_delegation = message.minSelfDelegation;
    obj.delegator_address = message.delegatorAddress;
    obj.validator_address = message.validatorAddress;
    obj.pubkey = message.pubkey ? {
      typeUrl: "/cosmos.crypto.secp256k1.PubKey",
      value: fromBase64(decodeBech32Pubkey(message.pubkey).value)
    } : undefined;
    obj.value = message.value ? Coin.toAmino(message.value) : undefined;
    return obj;
  },
```

This causes the following error:
```
Argument of type 'Any' is not assignable to parameter of type 'string'.ts(2345)
(property) MsgCreateValidator.pubkey: Any
```

`decodeBech32Pubkey` takes a string but `message.pubkey` is an `Any`.

**Comment by NoahSaso** [comment]
When I fix the `fromAmino` function, I get another `undefined` issue:
```ts
  fromAmino(object: ValidatorAmino): Validator {
    return {
      operatorAddress: object.operator_address,
      consensusPubkey: {
        typeUrl: "tendermint/PubKeySecp256k1",
        value: object.consensus_pubkey.value
      },
```

```
Type error: Object is possibly 'undefined'.
```

`object.consensus_pubkey` can be undefined since I have `allowUndefinedTypes` set to `true`, but that is not handled here. Here is my current workaround but there should be something better I think:

```
      consensusPubkey: {
        typeUrl: "tendermint/PubKeySecp256k1",
        value: object.consensus_pubkey?.value ?? new Uint8Array()
      },
```

**Comment by Zetazzz** [comment]
Hi, Thank you so much for the feedback. We just fixed this issue in the latest version 0.105
